### ISSUE TITLE
bootstrap modal not working properly on mobile

### ISSUE THREAD

**Issue by msherax** [twbs/bootstrap#21546]
Hi, i am using bootstrap v3.3.7 on my project. the issue i am facing is that the modal is not being displayed as per required on mobile screen. the modal is fixed the the top of the page with fade screen. it doesn't close as well. 
i have tried stack overflow, i found this solution.
http://stackoverflow.com/questions/10437151/twitter-bootstrap-modal-on-mobile-devices
but unfortunately the suggested solution is not helping me. 

Can somebody help me please that how i can fix this issue.

**Comment by msherax** [comment]
Here are the screenshots

![small-screen](https://cloud.githubusercontent.com/assets/7464940/21677610/b80329e4-d35c-11e6-9321-78ff7eb30dd9.png)
![large-screen](https://cloud.githubusercontent.com/assets/7464940/21677611/b8071cac-d35c-11e6-8743-89355abc10b3.png)

https://github.com/twbs/bootstrap/issues/3230
this link is also suggesting a solution but i am unable to follow it as its talking about bootstrap-responsive.css and i cant see this file in v3.3.7,

**Comment by mdo** [comment]
Bootstrap 3 is essentially in maintenance mode as we focus on working towards a stable v4. As such, we're only accepting changes to v3's code on a case-by-case basis, usually only for critical bug fixes or docs improvements.

---

_This is a saved reply._

**Comment by quinhone** [comment]
Version 4 also has this problem

**Comment by Mamadgi-Aishwarya** [comment]
Hey.I am facing the same problem. Did u find a solution?

**Comment by UjwalSadharia** [comment]
Hello everyone i got the solution for this probilem all you have to do is just paste you modal HTML just above **</body>** tag and its done. Thank me later

**Comment by UjwalSadharia** [comment]
Hello everyone i got the solution for this probilem all you have to do is just paste you modal HTML just above </body> tag and its done. Thank me later

**Comment by i0nu7z** [comment]
> Hello everyone I got the solution for this problem all you have to do is just paste you modal HTML just above tag and its done. Thank me later

Kudos @UjwalSadharia ! I was keep looking for few hours why the modal does not show up in mobile sized screens... I didn't really do like you said but you planted the idea. So instead of placing the code for my modal outside HTML Tags (which I highly not recommend), I just placed the modal right after the </body> tag (outside the body, under). Worked like a charm. Thanks for answering such an old question (I compared the dates).
